Source/bmalloc/libpas/src/libpas/pas_debug_heap.h

```diff
@@ -38,12 +38,14 @@ PAS_BEGIN_EXTERN_C;
 
 #if PAS_BMALLOC
 
+#include "BExport.h"
```

Comment:
bmalloc includes libpas. But libpas does not include bmalloc. Thus, we cannot include bmalloc header from libpas.

Reply:
libpas already does this:
https://github.com/WebKit/WebKit/blob/cffbf9134817e83c106e9303a9bd59ced78f7335/Source/bmalloc/libpas/src/libpas/pas_config.h#L33-L34
https://github.com/WebKit/WebKit/blob/cffbf9134817e83c106e9303a9bd59ced78f7335/Source/bmalloc/libpas/src/libpas/jit_heap_config.c#L43-L44

Since these includes are always guarded by `#if PAS_BMALLOC`, I think it's safe. In Xcode, these headers are always available (because libpas and bmalloc are the same project).

If we should avoid adding additional imports like this, I could inline the macro. Or maybe these declarations can be moved around. What do you think?

Reply:
Please add a FIXME about this.